component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentFamilyMeta.java

```diff
@@ -126,6 +135,26 @@ public ComponentBundle findBundle(final ClassLoader loader, final Locale locale)
                 }
             });
         }
+        
+        /**
+         * Returns a {@link Collection} of input flows names of this Component
+         * Should be overridden in inheritors
+         * 
+         * @return input flows names collection
+         */
+        public Collection<String> getInputFlows() {
```

Comment:
Abstract if 100% dependent from the child - no loose default ;)

Reply:
Moved from BaseMeta to FlowsFactory class. Made it abstract in parent class